src/main/java/com/algosenpai/app/user/README.md

```diff
@@ -0,0 +1,3 @@
+### User
```

Comment:
Change the name to something more descriptive like userStats/ userInfo?

Reply:
ok